src/components/businesses_search/get_distance.js

```diff
@@ -0,0 +1,20 @@
+
+export function getDistance(lat_t, lon_t, lat_b, lon_b) {
```

Comment:
Maybe it would be better to send just coordinates and let the database to filter results, it will be a way faster

Reply:
The database sends back all businesses in a "target area". This function only calculates the distance from a target point to a single business. DB store everything in degrees and frontend do all necessary calculations in meters.